pages/calendar.vue

```diff
@@ -4,14 +4,14 @@
     <div class="ui middle aligned stackable grid container">
       <div class="row">
         <div class="eight wide left column">
-          <h3 class="ui header">Meetings: Winter 2020</h3>
+          <h3 class="ui header">Meetings: Fall 2021</h3>
           <!-- <p> General meetings cover announcements, defensive/offensive demos, tech-talks, and guest speakers. We also host weekly practices for Defensive and Offensive competitions! Anyone is free to join. Email <b> cyberucr@gmail.com </b> for more info. </p>  -->
           <p> This Winter we're hosting biweekly intermediate-leveled General Meetings alongside defensive and offensive security workshop meetings. Download our Google calendar to yours! </p> 
```

Comment:
You fixed the header to say Fall, but need to fix the description to reflect that

Reply:
issue has been fixed

Reply:
Ty